code/obj/flock/structure/gnesisturret.dm

```diff
@@ -122,6 +123,17 @@
 							sleep(src.current_projectile.shot_delay)
 					shoot_projectile_ST_pixel_spread(src, current_projectile, target, 0, 0 , spread)
 
+	sapper_power()
+		if (!src.powered || !..())
+			return FALSE
+		src.accepts_sapper_power = FALSE
+		src.fluid_gen_amt *= 4
+		SPAWN(10 SECONDS)
+			if (src)
```

Comment:
I don't *think* it's possible for `src` to go null in a spawn unless something manually nulls it, but I'm fine with erring on the safe side